### DISCUSSION TITLE
Question : How to check which Autogluon version i have ?

### DISCUSSION THREAD

**Discussion by mohdfaaf** [autogluon/autogluon#1348] [Q&A] [answered]
All - i am a newbie , just joined github.. and would like to learn and apply ML stuff. 

I have managed to installed autogluon in my laptop which runs on windows...   i have any questions..  

How to i check which version of autogluon i have? 

Thanks

**Comment by Innixma** [comment, marked as answer]
You can do so via:

```
import autogluon.core
print(autogluon.core.__version__)
```

You can also check via command line with `pip freeze`.

**Reply by mohdfaaf** [reply]
Thank you!
